packages/xo-web/src/xo-app/host/tab-general.js

```diff
@@ -227,6 +241,27 @@ export default decorate([
             </Col>
           </Row>
         )}
+        <br />
+        {biosData !== undefined && (
+          <Row className='text-xs-center'>
+            <Col>
+              <Icon icon='bios-version' size={3} />
+              <p>
+                {_('currentBiosVersion', { version: biosData.currentBiosVersion })}{' '}
+                <Icon
+                  icon={biosData.isUpToDate ? 'success' : 'false'}
+                  className={biosData.isUpToDate ? 'text-success' : 'text-danger'}
+                />
+                <br />
+                {!biosData.isUpToDate && (
+                  <a href={biosData.biosLink} target='_blank' rel='noopener noreferrer'>
```

Comment:
`noreferrer` implies `noopener`.
```suggestion
                  <a href={biosData.biosLink} target='_blank' rel='noreferrer'>
```